src/components/SidePanel/SidePanel.tsx

```diff
@@ -172,19 +187,23 @@ class SidePanel extends React.Component<ISidePanelProps, ISidePanelState, {}> {
 
 	handleResize = ({ dX }: { dX: number }): void => {
 		const { startWidth } = this.state;
+		const position = startWidth + dX * (this.props.position === 'right' ? -1 : 1);
+		const adjustedWidth = !_.isNil(this.props.minWidth) && position < this.props.minWidth ? this.props.minWidth
```

Comment:
Take a look at https://lodash.com/docs/4.17.15#clamp and see if it would simplify this.

Reply:
ah thanks. I thought there was a lodash implementation for this but couldn't find what it was called.